tools/pythonpkg/tests/fast/api/test_native_tz.py

```diff
@@ -0,0 +1,79 @@
+import duckdb
+import datetime
+import pytz
+import os
+import pandas as pd
+import pytest
+
+filename = os.path.join(os.path.dirname(os.path.realpath(__file__)),'..','data','tz.parquet')
+
+def test_native_python_timestamp_timezone():
+	con = duckdb.connect('')
+	con.execute("SET timezone='America/Los_Angeles';")
+	res = con.execute(f"select TimeRecStart as tz  from '{filename}'").fetchone()
+	assert res[0].hour == 14 and res[0].minute == 52
+	assert res[0].tzinfo.zone == 'America/Los_Angeles'
+
+	res = con.execute(f"select TimeRecStart as tz  from '{filename}'").fetchall()[0]
+	assert res[0].hour == 14 and res[0].minute == 52
+	assert res[0].tzinfo.zone == 'America/Los_Angeles'
+
+	res = con.execute(f"select TimeRecStart as tz  from '{filename}'").fetchmany(1)[0]
+	assert res[0].hour == 14 and res[0].minute == 52
+	assert res[0].tzinfo.zone == 'America/Los_Angeles'
+
+	con.execute("SET timezone='UTC';")
+	res = con.execute(f"select TimeRecStart as tz  from '{filename}'").fetchone()
+	assert res[0].hour == 21 and res[0].minute == 52
+	assert res[0].tzinfo.zone == 'UTC'
+
+def test_native_python_time_timezone():
+	con = duckdb.connect('')
+	con.execute("SET timezone='America/Los_Angeles';")
+	res = con.execute(f"select TimeRecStart::TIMETZ as tz  from '{filename}'").fetchone()
```

Comment:
Is this intended to be supported?

Reply:
I agree. The reason I implemented it is that initially, I thought the functionality already existed "properly" on our other integrations. I'll remove it to avoid confusion and will just throw instead.